library/debug.c

```diff
@@ -85,7 +87,7 @@ void mbedtls_debug_print_msg( const mbedtls_ssl_context *ssl, int level,
 {
     va_list argp;
     char str[DEBUG_BUF_SIZE];
-    int ret;
+    int ret = MBEDTLS_ERR_ERROR_CORRUPTION_DETECTED;
```

Comment:
In commit message for "Initialize return values to an error", we have "... makes the library more robust.".

What does this make the library more robust against? Programmer errors? Please expand the commit message description with the dimensions within which there is increased robustness.